### ISSUE TITLE
Some questions about the Open-book Knowledge-store

### ISSUE THREAD

**Issue by sone225** [zjunlp/PromptKG#28]
Dear researchers,

I have been reading the research paper "Decoupling Knowledge from Memorization: Retrieval-augmented Prompt Learning" and I have a few questions regarding the Open-book Knowledge-store mentioned in the paper. I would like to know how this knowledge store is built and if this method can be applied to the field of computer vision.

Could you kindly provide me with more information or resources regarding these topics? Any help or insights would be greatly appreciated.

Thank you for your time and consideration. I look forward to hearing back from you soon.

Sincerely,

sone

**Comment by njcx-ai** [comment]
Dear sone:

Thanks for your attention to our research. Specifically, the knowledge store is constructed with key and value, where the key is the representation of instance and the value is it's corresponding label. Actually,  this method can be friendly applied to the field of image classification.

Sincerely,
Xiang Chen

**Comment by sone225** [comment]
Dear respected researchers,

Thank you for your previous response. I still have some questions regarding the Open-book Knowledge-store mentioned in your paper. I am wondering if the Open-book Knowledge-store is generated from the training dataset? Additionally, if the total number of images in the training dataset is only 500, and the number of images in some classes is even as small as 8, can an Open-book Knowledge-store still be constructed?

I appreciate your time and insights on this matter. Thank you for your help.

Sincerely,
sone

**Comment by njcx-ai** [comment]
Dear sone:

Actually, our experiments in this paper mainly involve few-shot settings where the number of each class is both as small as 4 or 16. The Open-book Knowledge-store can be successfully constructed.

Sincerely,
Xiang Chen
